build.gradle

```diff
@@ -1,10 +1,13 @@
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
 plugins {
   id 'application'
   id 'org.springframework.boot' version '2.1.6.RELEASE'
 }
 
 group = 'uk.gov.hmcts.reform'
-version = '1.0.0'
+version = "1.0.0-" + LocalDateTime.now().format(DateTimeFormatter.ofPattern("ddMMYYYY-HHMM"))
```

Comment:
would suggest the standard of `YYYYMMdd` instead of the other way around

Reply:
updated the date format